types/fuzzaldrin-plus/index.d.ts

```diff
@@ -1,29 +1,53 @@
-// Type definitions for fuzzaldrin-plus
+// Type definitions for fuzzaldrin-plus 0.6.0
 // Project: https://github.com/jeancroy/fuzzaldrin-plus/
-// Definitions by: Jean Christophe Roy <https://github.com/jeancroy>, Jason Killian <https://github.com/jkillian>
+// Definitions by:  Jean Christophe Roy <https://github.com/jeancroy>,
+//                  Jason Killian <https://github.com/jkillian>
+//                  Ronald Rey <https://github.com/reyronald>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export as namespace fuzzaldrin;
+export as namespace fuzzaldrin
 
-export interface IQueryOptions {
-    pathSeparator?: string;
-    optCharRegEx?: RegExp;
+export class Query {
+    $$__internal: Symbol
 }
 
-export interface IScoringOptions extends IQueryOptions {
-    allowErrors?: boolean;
-    isPath?: boolean;
-    useExtensionBonus?: boolean;
+export interface IFuzzaldrinPlusOptions {
```

Comment:
Is there a compelling reason to include `Fuzzaldrin` in all these interface names? 

```ts
import * as fuzzAldrin from 'fuzzaldrin-plus'

...

const options: fuzzAldrin.IFuzzaldrinPlusOptions = {
  // ...
}
```

Without these, you can make it a bit more terse:

```ts
import * as fuzzAldrin from 'fuzzaldrin-plus'

...

const options: fuzzAldrin.IOptions = {
  // ...
}

Reply:
Not really, just thought that if someone might want to import the type directly it could collide with other locally named `IOptions` type since it's such a common name. But in that case they can just alias it. So yeah, will change!

Reply:
Yeah, I'd leave that up to the consumer as they have full control here if there is a clash:

```ts
import { score, IOptions as IFuzzAldrinOptions } from 'fuzzaldrin-plus'
```